karma-test-shim.js

```diff
@@ -29,6 +29,9 @@ System.config({
     '@angular/http': 'npm:@angular/http/bundles/http.umd.js',
     '@angular/router': 'npm:@angular/router/bundles/router.umd.js',
     '@angular/forms': 'npm:@angular/forms/bundles/forms.umd.js',
+    '@angular/platform-browser/animations': 'npm:@angular/platform-browser/bundles/platform-browser-animations.umd.js',
+    '@angular/animations/browser': 'npm:@angular/animations/bundles/animations-browser.umd.js',
+    '@angular/animations': 'npm:@angular/animations/bundles/animations.umd.js',
```

Comment:
Do we need all those imports for tests, given that we are using `NoopAnimationsModule`? (Sorry for lame questions, learning about animation-related packages as I'm reviewing this PR :-))

Reply:
I believe so, both the `NoopAnimationsModule` and the `BrowserAnimationsModule` are exported from the same file